app/config/default.json

```diff
@@ -3,6 +3,7 @@
   "minimumCoreVersion": "4.0.0-rc.4",
   "protocol": "https",
   "host": "localhost",
+  "host_path": "",
```

Comment:
@nthfloor2 All of the other keys are named using camel case. We should stick to that convention for this one as well.